app/models/manageiq/providers/kubernetes/container_manager/metrics_capture.rb

```diff
@@ -1,10 +1,57 @@
+require 'hawkularclient'
+
 module ManageIQ::Providers
   class Kubernetes::ContainerManager::MetricsCapture < BaseManager::MetricsCapture
+    require_nested :CaptureContext
+
+    INTERVAL   = 20 # seconds
```

Comment:
better to rename the constant to include the word seconds rather than the comment - so it will be clear wherever it's used, not only where it's defined.
btw, also why not define 20.seconds? afaik the language allows it